### PR TITLE
NON-193: Non-associations list page

### PR DESCRIPTION
**Pull request by ushkarev**
Integrates the new "view" page into the user journeys, but the "list" page still only shows one table of non-associations. I.e. the list is still not split by establishment.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
         "@typescript-eslint/no-use-before-define": 0,
         "class-methods-use-this": 0,
         "no-useless-constructor": 0,
+        "quotes": ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": false }],
```
Comment (ushkarev) [inline .eslintrc.json:42]: template strings are magical, so it's best to only use them when necessary. as it happens, `allowTemplateLiterals` being false only forbids template literals when there are no variables inside

File: .eslintrc.json [hunk 2]
```diff
@@ -111,6 +112,7 @@
       }
     ],
     "no-use-before-define": 0,
+    "no-restricted-syntax": ["error", "ForInStatement", "LabeledStatement", "WithStatement"],
```
Comment (ushkarev) [inline .eslintrc.json:115]: this project is for the server-side, we can use `for (const a of b) {…}` because it’s 2023

File: cypress.config.ts [hunk 3]
```diff
@@ -17,8 +17,11 @@ export default defineConfig({
   reporterOptions: {
     configFile: 'reporter-config.json',
   },
+  video: false,
   videoUploadOnPasses: false,
   taskTimeout: 60000,
+  viewportWidth: 1200,
+  viewportHeight: 850,
```
Comment (ushkarev) [inline cypress.config.ts:24]: some GDS components change shape on smaller screens and cypress’ default is really small

File: integration_tests/mockApis/nonAssociationsApi.ts [hunk 4]
```diff
@@ -31,11 +31,12 @@ export default {
...
   },
 
+  /**
+   * Stub the list of non-associations for David Jones
+   */
   stubListNonAssociations({
-    prisonerNumber = davidJones.prisonerNumber,
     returning = 'twoOpen',
   }: {
-    prisonerNumber?: string
```
Comment (xoen) [inline integration_tests/mockApis/nonAssociationsApi.ts:38]: We don't need to stub other prisoners non-associations in any of the tests? (if you removed it clearly not).

I guess the result made only sense for `davidJones` anyway so pass something else could have lead to returning data that didn't make much sense anyway.

Comment (ushkarev) [reply integration_tests/mockApis/nonAssociationsApi.ts:38]: >the result made only sense for davidJones anyway
bingo!

**Review by xoen — APPROVED** [review]

**Pull request merged**